deployment/src/main/java/io/quarkiverse/quinoa/deployment/packagemanager/FrameworkType.java

```diff
@@ -28,6 +28,7 @@ public enum FrameworkType {
     REACT("build", "start", 3000, Set.of("react-scripts", "react-app-rewired", "craco")),
     VUE_LEGACY("dist", "serve", 3000, Set.of("vue-cli-service")),
     VITE("dist", "dev", 5173, Set.of("vite")),
+    SolidJS("dist", "dev", 3001, Set.of("vite")),
```

Comment:
I am not sure this will work because now two different enums will resolve with "vite" and it loops over them in order so first one will win.

Reply:
have you any suggetion @melloware @ia3andy

Reply:
Yeah let me do some thinking. I think one thing we could do is if it is something like "vite" then check the actual dependencies and look for "solidjs" vs "vue" etc.

Reply:
if you want this PR to work you should have some way to detect the solidjs framework (instead of vite)